backend/.env

```diff
@@ -0,0 +1,10 @@
+PGHOST=localhost
+PGPORT=5000
+PGUSER=postgres
+PGPASSWORD=123
+PGDATABASE=dev
+PGSSLMODE=disable
+
+EmailFrom= webbforumm@gmail.com
+EmailFromSMTPPw=kdlavcphfzutdgbv
```

Comment:
We should consider .gitignore-ing .env files. Maybe put a .env.example or something

Reply:
Okay will add it to gitignore